lib/StripeResource.js

```diff
@@ -598,11 +518,9 @@ StripeResource.prototype = {
           options.headers,
           options.settings
         );
-
-        makeRequest(apiVersion, headers);
+        makeRequest(apiVersion, headers, 0);
```

Comment:
This 0 is an addition- I remember a comment from a previous revision, but it seems to have been wiped from GH.